flask_appbuilder/templates/appbuilder/general/lib.html

```diff
@@ -75,7 +75,11 @@
 		{% for action_key in actions %}
         {% set action = actions.get(action_key) %}
 				$('.{{action.name}}_menu_item').on('click', function(){
-				    return modelActions.execute_multiple('{{action.name}}','{{action.confirmation}}');
+                    {% if confirmation %}
```

Comment:
indentation looks really bad here, could you take the chance and fix it also?

Reply:
In the file there was tabs and spaces before my editing. I've tried to keep indentation using tabs...
I will replace all tabs with spaces to keep indentation coherent with any editor.

Reply:
Thank you!